packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -163,6 +171,16 @@ export default class TooltipAnchor extends React.Component<Props, State> {
         this._updateActiveState(false, this._focused);
     };
 
+    _handleKeyUp = (e: KeyboardEvent) => {
+        const escape = 27;
+        const keyCode = e.which || e.keyCode;
+        if (keyCode === escape && this.state.active) {
+            e.preventDefault();
+            e.stopImmediatePropagation();
+            this._updateActiveState(false, false);
```

Comment:
This is to prevent the modal from also closing.  Can you add a comment about as to why we do `preventDefault` and `stopImmediatePropagation` here?  Also, why `stopImmediatePropagation` and not `stopPropagation`?

Reply:
Sure.